### ISSUE TITLE
Security records photo doesn't match the person

### ISSUE THREAD

**Issue by 00aquir** [yogstation13/Yogstation#9349]
The person (me) on the photo, checked on security HUD or sec records, is different than the actual one (hair, even skin color). The photo showed not the real thing, but what was visible in the character settings tab / body, when it was set to "random" before.

**Comment by redmoogle** [comment]
Sure they didnt mulligan?

**Comment by 00aquir** [comment]
It was me, so no. It's like this: https://github.com/yogstation13/Yogstation/issues/2892#issuecomment-476221086 And even the skin color can be different, right from the start.

**Comment by Bibby0110** [comment]
Yeah random doesn't work with the mugshots since they're generated based on your preferences and not based on your actual mob at the start
